cmd/manager/main.go

```diff
@@ -233,11 +244,6 @@ func Command() *cobra.Command {
 		DefaultWebhookName,
 		"Name of the Kubernetes ValidatingWebhookConfiguration resource. Only used when enable-webhook is true.",
 	)
```

Comment:
I was also expecting a flag to control the default timeout for ES API calls (which can also be overridden per cluster).